hooks/msghooks.py

```diff
@@ -126,7 +129,7 @@ def receive_message():
                 else:
                     return f"{reply} No, you sussy baka."
             else:
-                if random.random() <= 0.5 or message["user_id"] == 513493:
+                if random.random() <= 0.5 or message["user_id"] == 513493 or message["user_id"] == 354515:
```

Comment:
Again, I don't like the magic numbers or the repetition of `message[userid] ==`, I think you should use something like `in FLAG_HATER_LIST` instead.